frontend/src/Components/Charts/Searches/Searches.js

```diff
@@ -188,7 +191,8 @@ function Searches() {
         <ChartHeader
           chartTitle="Searches By Count"
           handleViewData={handleViewCountData}
-          handleShareGraph={handleShareCountGraph}
+          id={BY_COUNT_ANCHOR_ID}
```

Comment:
I see these `id`'s in the DOM, but the page doesn't scroll down to them when loading a URL with an anchor